tpch/tests/queries_test.py

```diff
@@ -4,20 +4,22 @@
 import sys
 from pathlib import Path
 
+import pytest
 
-def test_execute_scripts() -> None:
-    root = Path(__file__).resolve().parent.parent
-    # directory containing all the queries
-    execute_dir = root / "execute"
+ROOT_PATH = Path(__file__).resolve().parent.parent
+# Directory containing all the query scripts
+QUERIES_DIR = ROOT_PATH / "queries"
 
-    for script_path in execute_dir.glob("q[1-9]*.py"):
-        print(f"executing query {script_path.stem}")  # noqa: T201
-        result = subprocess.run(  # noqa: S603
-            [sys.executable, "-m", f"execute.{script_path.stem}"],
-            capture_output=True,
-            text=True,
-            check=False,
-        )
-        assert (
-            result.returncode == 0
-        ), f"Script {script_path} failed with error: {result.stderr}"
+
+@pytest.mark.parametrize("query_path", QUERIES_DIR.glob("q[1-9]*.py"))
```

Comment:
if this PR doesn't get merged, I think that at least we should have this parametrised :)

Reply:
+1